components/cdc/src/service.rs

```diff
@@ -405,6 +409,7 @@ impl Service {
             channel(CDC_CHANNLE_CAPACITY, self.memory_quota.clone());
         let conn = Conn::new(event_sink, ctx.peer());
         let conn_id = conn.get_id();
+        event_drain.set_conn_id(conn_id);
```

Comment:
How about creating conn_id separately? 

```suggestion
        let conn_id = ConnId::default();
        let (event_sink, mut event_drain) =
            channel(conn_id, CDC_CHANNLE_CAPACITY, self.memory_quota.clone());
        let conn = Conn::new(conn_id, event_sink, ctx.peer());
```